anaDACScan.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+"""
+anaDACScan
+==============
+"""
+import ROOT as r
+
+from gempython.gemplotting.utils.anautilities import parseCalFile
+from gempython.utils.nesteddict import nesteddict
+from gempython.gemplotting.utils.anautilities import make3x8Canvas
+
+import argparse
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description='Arguments to supply to anaDACScan.py')
+
+    parser.add_argument('infilename', type=str, help="Filename from which input information is read", metavar='infilename')
+    parser.add_argument("calFileFile", type=str, help="File specifying which calFile to use for each OH. Format: OH1 filenameOH1 <newline> OH2 filenameOH2 <newline> etc", metavar="calFileFile")
```

Comment:
per our discussion make this optional and if not provided have a look up in `DATA_PATH` per the example described [here](https://mattermost.web.cern.ch/cms-gem-daq/pl/o5aknmyn6tdgun6mzw1gmza44c).